OpenRA.Mods.Common/Traits/Player/DeveloperMode.cs

```diff
@@ -161,10 +162,28 @@ public void ResolveOrder(Actor self, Order order)
 				case "DevGiveCash":
 				{
 					var amount = order.ExtraData != 0 ? (int)order.ExtraData : info.Cash;
+
+					var playerName = order.Player.PlayerName;
+					Game.Debug("Giving {0} credits to player {1}.", order.ExtraData == 0 ? "cheat default" : amount.ToString(CultureInfo.InvariantCulture), playerName);
```

Comment:
This can simplify down to `Game.Debug("Giving {0} credits to player {1}.", amount, order.Player.PlayerName);` (and then remove the separate `playerName` variable).